modules/apps/portal-security/portal-security-ldap-impl/src/main/java/com/liferay/portal/security/ldap/internal/exportimport/LDAPUserExporterImpl.java

```diff
@@ -346,6 +346,12 @@ public void exportUser(
 
 			ModificationItem[] modificationItems = modifications.getItems();
 
+			if (userBinding == null) {
+				userBinding = _safePortalLDAP.getUser(
+					ldapServerId, companyId, user.getScreenName(),
+					user.getEmailAddress(), false, false);
+			}
+
```

Comment:
Can you help me why this part is required?

Reply:
Hi Oliver, 
addUser, in line 312, return a null Binding object in case "Import Search Filter" include an attribute because attributes will be included in LDAP later, in line 358, calling to  modifyAttributes method.